camelot/view/controls/editors/one2manyeditor.py

```diff
@@ -131,22 +130,11 @@ def set_right_toolbar_actions(self, action_routes):
             self.layout().addWidget(toolbar)
             # set field attributes might have been called before the
             # toolbar was created
-            self.update_action_status()
+            #self.update_action_status()
```

Comment:
set_right_toolbar_actions w in onze code enkel aangeroepen vanuit de constructor hierboven. Deze update_actions_status hebben we hier dus niet meer nodig?

Reply:
denk ik idd, in principe zijn nu de actions bepaald al bij constructie vd editor, daar waar dat vroeger 'dynamischer' was